src/backend/distributed/relay/relay_event_utility.c

```diff
@@ -274,7 +274,8 @@ RelayEventExtendNames(Node *parseTree, char *schemaName, uint64 shardId)
 				/* prefix with schema name if it is not added already */
 				if (relationSchemaNameValue == NULL)
 				{
-					relationNameList = lcons(makeString(schemaName), relationNameList);
+					Value *schemaNameValue = makeString(pstrdup(schemaName));
```

Comment:
We generally don't call functions as a parameter, instead can we do something like this;
char *schemaNameCopy = pstrdup(schemaName)
Value *schemaNameValue = makeString(schemaNameCopy);